Hunting Queries/BehaviorAnalytics/Anomalous Code Execution.yaml

```diff
@@ -0,0 +1,18 @@
+eid: f8ab176c-1f3f-4cb5-8dc1-f50d30bcae0d
+name: Anomalous Code Execution
+description: |
+    'Adversaries may abuse command and script interpreters to execute commands, scripts, or binaries. These interfaces and languages provide ways of interacting with computer systems and are a common feature across many different platforms. APT19, for example, used PowerShell commands to execute payloads. The query below generates an output of all users performing an "action" operation regarding "runCommand" in virtual machines, where one or more features of the activitiy deviates from the user, his peers or the tenant profile.'
+requiredDataConnectors:
+  - connectorId: BehaviorAnalytics
+    dataTypes:
+        - BehaviorAnalytics
+tactics:
+  - Execution
+relevantTechniques:
+  - T1059
+query: |
+    let operations = dynamic(['Run Command on Virtual Machine']);
+    BehaviorAnalytics
+    | where ActionType in(operations)
+    | where ActivityInsights contains "True"
```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.

Reply:
done